.dictionary

```diff
@@ -97,6 +98,7 @@ ios
 janerik
 ktlint
 lang
+lfloor
```

Comment:
The aspell filter for markdown doesn't correctly ignore math chunks.  It's ok to just throw them in the dictionary for now, but it might get annoying if we try to make our documentation look like an academic paper.